pywbem/cim_operations.py

```diff
@@ -636,6 +658,26 @@ def __init__(self, url, creds=None, default_namespace=DEFAULT_NAMESPACE,
         self._last_operation_time = None
         self._last_server_response_time = None
 
+        # logging
+        self.enable_logging = enable_logging
+
+        if self.enable_logging:
+            if enable_logging == 'all':
+                log_limit = None
+            elif enable_logging == 'min':
+                log_limit = DEFAULT_MAX_LOG_ENTRY_SIZE
+            elif isinstance(enable_logging, six.integer_types) and \
+                    enable_logging > 0:
+                log_limit = enable_logging
```

Comment:
The possibility to be an integer is missing in the description of the parameter.

Reply:
DONE